website/server/libs/inbox/index.js

```diff
@@ -99,9 +98,32 @@ export async function listConversations (owner) {
 
   const conversationsList = await query.exec();
 
+  const usersQuery = {
+    _id: {$in: conversationsList.map(c => c._id) },
+  };
+
+  const usersAr = await User.find(usersQuery,  {
+    _id: 1,
```

Comment:
why 1,2,3..? I think 1 means take the filed, 0 don't so values over 1 don't do anything

Reply:
woops, guess I misread an example for the projection^^, tested this now again with just `1` works, too bad there isn't a warning/hint when I used 2 and higher :/

fixing as I write this